redirects.txt

```diff
@@ -107,4 +107,10 @@ inventory/management/adjustment/min_stock_rule_vs_mto.rst inventory/management/r
 inventory/management/adjustment/virtual_warehouses.rst inventory/management/replenishment_strategies/virtual_warehouses.rst 	 	# adjustment/* --> replenishment_strategies/* (#424)
 inventory/settings/warehouses.rst inventory/management/warehouses.rst 																# settings/* --> management/* (#424)
 inventory/settings/warehouses/difference_warehouse_location.rst inventory/management/warehouses/difference_warehouse_location.rst   # settings/* --> management/* (#424)
-inventory/settings/warehouses/location_creation.rst inventory/management/warehouses/location_creation.rst 							# settings/* --> management/* (#424)
\ No newline at end of file
+inventory/settings/warehouses/location_creation.rst inventory/management/warehouses/location_creation.rst 							# settings/* --> management/* (#424)
+
+helpdesk/getting_started.rst helpdesk/overview/getting_started.rst                               #helpdesk/* -> overview/* (#565)
+helpdesk/after_sales.rst helpdesk/advanced/after_sales.rst                                       #helpdesk/* -> advanced/* (#565)
+helpdesk/close_tickets.rst helpdesk/advanced/close_tickets.rst                                   #helpdesk/* -> advanced/* (#565)
+helpdesk/invoice_time.rst helpdesk/timesheet_and_invoice/invoice_time.rst                        #helpdesk/* -> timesheet_and_invoice/* (#565)
+helpdesk/reinvoice_from_project.rst helpdesk/timesheet_and_invoice/reinvoice_from_project.rst    #helpdesk/* -> timesheet_and_invoice/* (#565)
```

Comment:
If it's not possible to summarize the changes in the comment, just leave the PR id : `# (#565)`
FYI, `helpdesk/* -> overview/*` is read as "Everything in `helpdesk/` moves to `overview/`.

Reply:
but it is per line..no?! So, the content (everything on that line) that was on helpdesk moved to overview.
I saw - what I interpreted to be the same - being done for accounting and inventory. No? Maybe I just got it wrong then.
-> Changed! Thanks!!

Reply:
Ah ok I see your point. So, no, the comment is not 'filtered' by the rule on the left. The comment just say why you did the change: "Because this file moved somewhere else".
That's really not important, just letting you know ;)